dbfit-java/core/src/main/java/dbfit/util/DbConnectionProperties.java

```diff
@@ -26,18 +26,28 @@ private static DbConnectionProperties createFrom(Map<String, String> map) {
             String key = entry.getKey();
             String val = entry.getValue();
 
-            if ("username".equals(key)) {
-                props.Username = val;
-            } else if ("password".equals(key)) {
-                props.Password = val;
-            } else if ("service".equals(key)) {
-                props.Service = val;
-            } else if ("database".equals(key)) {
-                props.DbName = val;
-            } else if ("connection-string".equals(key)) {
-                props.FullConnectionString = val;
-            } else {
-                throw new UnsupportedOperationException(
+            switch (key) {
+                case "username":
+                    props.Username = val;
+                    break;
+                case "password":
+                    props.Password = val;
+                    break;
+                case "service":
+                    props.Service = val;
+                    break;
+                case "database":
+                    props.DbName = val;
+                    break;
+                case "connection-string":
+                    props.FullConnectionString = val;
+                    break;
+                case "admin-username":
```

Comment:
Good questions. This is the area I doubted most in this PR.

I guess another properties file for DBA credentials could be used with the only downside that we'd need to likely duplicate the database service and database names.